kong/plugins/prometheus/exporter.lua

```diff
@@ -41,21 +42,18 @@ local function init()
   prometheus = require("kong.plugins.prometheus.prometheus").init(shm, "kong_")
 
   -- global metrics
-  if kong_subsystem == "http" then
-    metrics.connections = prometheus:gauge("nginx_http_current_connections",
-      "Number of HTTP connections",
-      {"state"})
-  else
-    metrics.connections = prometheus:gauge("nginx_stream_current_connections",
-      "Number of Stream connections",
-      {"state"})
-  end
+  metrics.connections = prometheus:gauge("nginx_current_connections",
```

Comment:
```suggestion
  metrics.connections = prometheus:gauge("nginx_connections",
```
The `state` label can be `total`, `handled` or `accepted`, `active`, `waiting`, `reading`, `writing`. This is a mix of guages and couners. It is best to not put `current_` in here since that has potential for added confusion.

Reply:
I don't feel strongly on wording/naming here. @sichvoge do you?

Also I don't see any counters here? Am I missing some context?

Reply:
The total, handled, and accepted are counters (they only go up) but are implemented as guages.

Reply:
While we are making breking changes to names, I would suggest to do https://github.com/Kong/kong/pull/8766 (as suggested by #7680) and make the `total` a seperate metrics, like `nginx_requests_total`.

Reply:
@hfukada I think you are actually doing that already in line 46 right?